query.mjs

```diff
@@ -28,6 +26,17 @@ const includeHasCitationcff = async (url) => {
 }
 
 
+const includeUsesWorkflows = async (url) => {
+
+    const [ owner, repo, ...unuseds ] = url.slice("https://github.com/".length).split('/');
+    const { data: { total_count: nworkflows } } = await octokit.rest.actions.listRepoWorkflows({
+        owner,
+        repo,
+    });
+    return nworkflows > nworkflows_minimum;
```

Comment:
The naming "minimum" implies that's the acceptable number of workflows, so I think this condition should be `>=`.

Reply:
done